src/app/sections/ProbeDetails.svelte

```diff
@@ -194,6 +210,13 @@ h2 {
             </div>
         </div>
         {/if}
+        <!--div class="drawer-section">
```

Comment:
should this be deleted?

Reply:
indeed...i should review my own diffs better